src/components/Header/index.js

```diff
@@ -203,6 +205,14 @@ export default class GordonHeader extends Component {
       }
     }
 
+    //Add to return statement when re-enabling work link
```

Comment:
When this is incorporated, will it need different versions for offline or guest 360 as the People Tab did?

Reply:
Good question, right now there is a guest view so that guests cannot access the timesheets page. It's a good idea for us to add the offline view too.